app/gui/src/model/suggestion_database/entry.rs

```diff
@@ -417,118 +574,69 @@ impl Entry {
 
 impl Entry {
     /// Create entry from the structure deserialized from the Language Server responses.
-    pub fn from_ls_entry(entry: language_server::types::SuggestionEntry) -> FallibleResult<Self> {
+    pub fn from_ls_entry(mut entry: language_server::types::SuggestionEntry) -> Self {
         use language_server::types::SuggestionEntry::*;
-        let this = match entry {
-            #[allow(unused)]
-            Type {
-                name,
-                module,
-                params,
-                documentation,
-                documentation_html,
-                documentation_sections,
-                ..
-            } => {
-                let referent_name = ReferentName::new(name.clone())?;
-                let mut module_name: module::QualifiedName = module.clone().try_into()?;
-                module_name.push_segment(referent_name);
-                let return_type = module_name.to_string();
-                Self {
-                    name,
-                    arguments: params,
-                    return_type,
-                    documentation_html: Self::make_html_docs(documentation, documentation_html),
-                    module: module.try_into()?,
-                    self_type: None,
-                    kind: Kind::Type,
-                    scope: Scope::Everywhere,
-                    icon_name: find_icon_name_in_doc_sections(&documentation_sections),
-                }
+
+        fn to_qualified_name(s: String) -> QualifiedName {
+            s.as_str().try_into().unwrap_or_else(|_| {
+                error!("Invalid qualified name received from the Engine in SuggestionEntry. Assuming an entinty in Standard.Base project");
```

Comment:
Typo in `Assuming an entinty`. Also, is it necessary to hard-code `Standard.Base` here?